src/main/java/executor/service/DIFactory/ObjectFactoryImpl.java

```diff
@@ -0,0 +1,95 @@
+package executor.service.DIFactory;
+
+import java.lang.reflect.*;
+import java.util.Map;
+import java.util.Set;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.function.Supplier;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.reflections.Reflections;
+
+public class ObjectFactoryImpl implements ObjectFactory {
```

Comment:
This DI does not inject fields.
![image](https://github.com/GFLCourses5/worker/assets/140727960/72fd6a1f-0dc0-4d51-9464-b2a2764ac539)

We also need to give recommendations on creating a default constructor in each class.